Update dependencies

CanerKaraca23 opened the pull request:
https://github.com/JingMatrix/LSPosed/pull/12#issuecomment-2323498077

CanerKaraca23 committed:
Update dependencies (#12)

JingMatrix committed:
Update fmt submodule

CanerKaraca23 committed:
Merge branch 'JingMatrix:master' into master

CanerKaraca23 committed:
Update

CanerKaraca23 commented:
@JingMatrix I think trying fixing mainswitchbar on another draft PR will be good since there is no issue in this PR and not includes mainswitchbar update and good to merge.

JingMatrix commented:
The dependent bot (included in this PR) will anyway always try to update the mainswitchbar version, we should fix it before the bot try to add new pull-requests.

JingMatrix committed:
chmod +x gradlew

CanerKaraca23 commented:
> The dependent bot (included in this PR) will anyway always try to update the mainswitchbar version, we should fix it before the bot try to add new pull-requests.

You can fix in dependabot's PR or simply ignore just like how LSPosed devs did. https://github.com/LSPosed/LSPosed/pull/1902

JingMatrix committed:
Change dependency bot frequence

For submodules, I will manage the update by myself.

Since github-actions update won't actually change the built app, we
don't need to check it so often.
